### ISSUE TITLE
Version 2023.3 - Release Notes Error - Grafana 8.5.15

### ISSUE THREAD

**Issue by bigmike-19** [uyuni-project/uyuni#6686]
In the release notes for version 2023.03, it states that Grafana was upgraded to 8.5.15. However, this upgrade happened in 2023.01. 
Grafana was upgraded to 8.5.20 in the 2023.03 release. Can we please get the release notes updated with the what fixes were with the 8.5.20 release?

Thanks, 
Mike

**Comment by juliogonzalez** [comment]
@bigmike-19 the release notes contain a link to the upstream changelog, but I am not sure why they don't have 8.5.20 there. Not sure if @witekest knows.

In any case, here goes the list, straight from the changelogs:
```
- Update to version 8.5.20:
  * Security: SVG: Add dompurify preprocessor step
    (bsc#1207749, CVE-2022-23552)
  * Security: Snapshots: Fix originalUrl spoof security issue
    (bsc#1207750, CVE-2022-39324)
  * Security: Omit error from http response 
  * Bug fix: Email and username trimming and invitation validation
 ```
I will update the release notes in the website, so they contain the changelogs, as it's clear upstream doesn't have the info.

The packages for the server UI will get the fix together with 2023.04

**Comment by juliogonzalez** [comment]
https://www.uyuni-project.org/doc/2023.03/release-notes-uyuni-server.html#_monitoring_grafana_update_to_8_5_20 fixed.

I left the link to the upstream release notes in case they restore 8.5.20

The WebUI will get the the release notes fixed together with 2023.04.

**Comment by bigmike-19** [comment]
Thank you for the information.
